src/xunit.console.netcore/project.lock.json

```diff
@@ -35,7 +35,7 @@
           "lib/dotnet/System.Collections.Concurrent.dll": {}
         }
       },
-      "System.Console/4.0.0-beta-23316": {
```

Comment:
Can you please fix the project.json version of this to 23316 for this project and the other? I want to avoid updating these versions and I want to start removing the * versions from them.